### PR TITLE
Update store-apps-company-portal-autopilot.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to reflect that the company portal updates will occur automatically to any assignments

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/memdocs/issues/2233

**Comment by PRMerger3** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@Erikre Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: memdocs/intune/apps/store-apps-company-portal-autopilot.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ To manage devices and install apps, your users can use the Company Portal app. Y
...
 
-You can choose to install the **Company Portal (Offline)** app using the steps below. The Company Portal app will be installed in device context when assigned to the Autopilot group and will be installed on the device before the user logs in. Offline apps are updated using Intune, whereas online apps are updated by the store. Use offline apps when you need to install and maintain a specific app version.
+You can choose to install the **Company Portal (Offline)** app using the steps below. The Company Portal app will be installed in device context when assigned to the Autopilot group and will be installed on the device before the user logs in. Offline apps are updated using Intune, whereas online apps are updated by the store. When synced from the Microsoft Store for Business and deployed as a Microsoft Store app, the Company Portal (Offline) updates will occur automatically to any required assignments. If maintaining a specific version of the Company Portal is required, please refer to [Manually add the Windows 10 company portal app](/mem/intune/apps/store-apps-company-portal-app).
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/store-apps-company-portal-autopilot.md:43]:
```suggestion
You can choose to install the **Company Portal (Offline)** app using the steps below. The Company Portal app will be installed in device context when assigned to the Autopilot group and will be installed on the device before the user logs in. Offline apps are updated using Intune, whereas online apps are updated by the store. When synced from the Microsoft Store for Business and deployed as a Microsoft Store app, the Company Portal (Offline) updates will occur automatically to any required assignments. If you need to maintain a specific version of the Company Portal, refer to [Manually add the Windows 10 company portal app](/mem/intune/apps/store-apps-company-portal-app) for details.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @msbemba, kindly commit the suggested changes. Many thanks.

**Comment by PRMerger13** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**